### PR TITLE
Update deployment-models-and-regions.txt

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Jira

- https://jira.mongodb.org/browse/DOCSP-NNNNN

### Staged Changes (Requires MongoDB Corp SSO)

- [Deployment Models and Regions](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/azure/manage-apps/deploy/deployment-models-and-regions/)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/manage-apps/deploy/deployment-models-and-regions.txt [hunk 1]
```diff
@@ -70,7 +70,15 @@ Cloud Deployment Regions
 - Sydney (``ap-southeast-2``)
 - Virginia (``us-east-1``)
 
-The following regions are available for :ref:`local deployment
+{+service+} supports deployment and writes in the following Azure regions:
+
+- ``EUROPE_WEST``
```
Comment (tsedgwick) [inline source/manage-apps/deploy/deployment-models-and-regions.txt:75]: Do you want this to align to what we show on the UI?

Screen Shot 2022-06-06 at 2 41 37 PM

Comment (nickpoindexter) [inline source/manage-apps/deploy/deployment-models-and-regions.txt:75]: [nit] We may want to use the Azure region name here to match the other lists on this page. 
```
westeurope
eastus2
southeastasia
eastasia
westus
```

**Review by nickpoindexter — CHANGES_REQUESTED** [review]
Just a small addition in case customers aren't familiar with the Atlas region name.

File: source/manage-apps/deploy/deployment-models-and-regions.txt [hunk 2]
```diff
@@ -70,7 +70,15 @@ Cloud Deployment Regions
...
+- ``US_EAST_2``
+- ``ASIA_SOUTH_EAST``
+- ``ASIA_EAST``
+- ``US_WEST``
```
Comment (nathan-contino-mongo) [inline source/manage-apps/deploy/deployment-models-and-regions.txt:79]:
```suggestion
- Virginia (``eastus2``)
- California (``westus``)
- Netherlands (``westeurope``)
- Singapore (``southeastasia``)
- Hong Kong (``eastasia``)
```

File: source/manage-apps/deploy/deployment-models-and-regions.txt (hunk 1 above)
Comment (nathan-contino-mongo) [reply source/manage-apps/deploy/deployment-models-and-regions.txt]: Updated to match this list, thank you!

Comment (nathan-contino-mongo) [reply source/manage-apps/deploy/deployment-models-and-regions.txt]: Updated to match Tim's list!

**Review by nickpoindexter — APPROVED** [review]

**Review by tsedgwick — APPROVED** [review]

**Pull request merged**